janitor/functions.py

```diff
@@ -29,20 +29,16 @@
 from sklearn.preprocessing import LabelEncoder
 
 from .errors import JanitorError
-from .utils import (
-    _clean_accounting_column,
-    _currency_column_to_numeric,
-    _replace_empty_string_with_none,
-    _replace_original_empty_string_with_none,
-    _strip_underscores,
-    check,
-    check_column,
-    deprecated_alias,
-)
+from .utils import (_clean_accounting_column, _currency_column_to_numeric,
+                    _replace_empty_string_with_none,
+                    _replace_original_empty_string_with_none,
+                    _strip_underscores, check, check_column, deprecated_alias,
+                    _grid_computation, _check_instance)
```

Comment:
Could you run a `black -l 79 .` on the source and push? That will solve the `isort` formatting issues. We default to `black` as the authoritative code formatter.